llvm/test/CodeGen/AMDGPU/GlobalISel/amdgpu-irtranslator.ll

```diff
@@ -1,3 +1,4 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
```

Comment:
This one is broken, it would need to use update_mir_test_checks. Also for the purposes of your patch I would just remove the amdgpu_kernel from the function